compiler/one-cmds/one-pack

```diff
@@ -73,6 +73,7 @@ def _make_model2nnpkg_cmd(driver_path, input_path, output_path):
     cmd = [os.path.expanduser(driver_path)]
     cmd.append('-o')
     cmd.append(os.path.expanduser(output_path))
+    cmd.append('-m')
```

Comment:
@hseok-oh , this change is result of interface change in model2nnpkg.
I would rather revise python version to provide backward compatibility.
Is there any possibility to do that before landing this?

Reply:
From yesterday, we are testing compiler commands on PR test (`nncc-release`). It may prevent breaking command's backward compatibility.